src/ui/Thumbnail/Thumbnail.ts

```diff
@@ -28,16 +28,21 @@ export class Thumbnail extends Component {
    * @componentOptions
    */
   static options: IThumbnailOptions = {
+
     /**
-     * Specifies the CSS class to use on the thumbnail image tag when a result
-     * has no thumbnail in the index.<br/>
-     * Default value: <code>coveo-no-thumbnail</code>
+     * Specifies the CSS class to use on the `img` tag that the Thumbnail component outputs when a result has no
+     * thumbnail in the index.
+     *
+     * Default value is `coveo-no-thumbnail`.
      */
     noThumbnailClass: ComponentOptions.buildStringOption({ defaultValue: 'coveo-no-thumbnail' }),
+
     /**
-     * Specifies if a clickable {@link ResultLink} is to be created around the Thumbnail.<br/>
-     * Uses all the same options as as {@link ResultLink} except <code>field</code><br/>
-     * Default value is <code>false</code>
+     * Specifies whether to create a clickable {@link ResultLink} around the Thumbnail.
+     *
+     * Uses all the same options as a {@link ResultLink} except for {@link ResultLink.options.field}.
```

Comment:
That would be a bug then.

Can you open one in jira ? We'll have a look at it.